test/packet/scripts/install.sh

```diff
@@ -10,68 +10,86 @@ VAGRANT_VERSION="2.2.4"
 PACKER_VERSION="1.3.5"
```

Comment:
Don't forget to push this upstream ;-)

Reply:
Is this upstream? https://github.com/cilium/packer-ci-build/blob/master/provision/ubuntu/install.sh